.ci/docker/sdk-linux/Dockerfile

```diff
@@ -1 +1 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.202
```

Comment:
Otherwise it will use 3.1.300 and it seems it's required to use 3.1.20x

```
docker run --rm -ti --name ecs-dotnet -u "$(id -u):$(id -g)" -v $(pwd):/src -w /src -e HOME=/tmp ${dockerTagName} ./build.sh release -s true

Welcome to .NET Core 3.1!
---------------------
SDK Version: 3.1.202
```

Otherwise it fails with:

```
....
${dockerTagName} ./build.sh release -s true                

Welcome to .NET Core 3.1!
---------------------
SDK Version: 3.1.300

...

[dll] Elastic.CommonSchema.BenchmarkDotNetExporter, Version=1.0.0.0, Culture=neutral, PublicKeyToken=069ca2728db333c1
[version] Assembly: 1.0.0.0
[version] AssemblyFile: 1.0.0.0
[version] Informational: 1.0.0+1c9c0fee96c97fc6bba34ba35923c86454d761d4
[version] lib/netstandard2.0/Elastic.CommonSchema.BenchmarkDotNetExporter.dll AssemblyFileVersion expected 1.5.2.0, actual: 1.0.0.0
:: validatepackages: Failed! Process exited with '1' "dotnet "nupkg-validator" "build/output/Elastic.CommonSchema.BenchmarkDotNetExporter.1.5.2-canary.0.4.7.nupkg" "-v" "1.5.2-canary.0.4.7+1c9c0fee96c97fc6bba34ba35923c86454d761d4" "-k" "069ca2728db333c1" "-t" "build/output"" (13.1 s)
:: ───────────────────────────────────────────
:: Duration        Outcome    Target          
:: ──────────────  ─────────  ────────────────
:: 13.08 s  20.8%  Failed!    validatepackages
```

In fact this is matching the version in the GitHub actions:
- https://github.com/elastic/ecs-dotnet/blob/master/.github/workflows/ci.yml#L30

_NOTE_: it uses 201 rather than 202 but it works like a charm locally and I hope that in the CI now